### ISSUE TITLE
Incorrect font rendering

### ISSUE THREAD

**Issue by alosarjos** [brave/brave-browser#12746]
## Description 
The computed font for rendering some texts is incorrectly calculated and using the wrong font.

## Steps to Reproduce

   1. Setup personalized fonts. I have used Cantarell for standard, Serif and Sans Serif and Source Code Pro for monospaced
   2. Access a Github file page, for example: https://github.com/brave/brave-browser/blob/master/package-lock.json
   3. Inspect the text for code.

## Actual result:
The font family for the text is `font-family:  SFMono-Regular,Consolas,Liberation Mono,Menlo,monospace;`. Since none of those fonts ( SFMono-Regular,Consolas,Liberation Mono,Menlo) is on the system, it should default to monospace and render the text with Source Code Pro but instead, its rendering the text with Cantarell font (Can check from the computed tab on the web inspector). If I remove these fonts except monospace, text is rendered with Source Code Pro.

TLDR: Non existing fonts on the system ( SFMono-Regular,Consolas,Liberation Mono,Menlo) are making the text be rendered with a different font, instead of using the monospace that is set at the end of the CSS.

## Expected result:

Compute the CSS to use the monospaced font of the system if none of the others are installed

## Reproduces how often: 
Easily reproduced

## Brave version (brave://version info)

Brave | 1.16.76 Chromium: 86.0.4240.198 (Build oficial) unknown (64 bits)
Revisión | d8a506935fc2273cfbac5e5b629d74917d9119c7-refs/branch-heads/4240@{#1431}
Sistema operativo | Linux

## Version/Channel Information:
- Can you reproduce this issue with the current release?  Yes
- Can you reproduce this issue with the beta channel?  Not Tested
- Can you reproduce this issue with the nightly channel? Not Tested

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? Yes

## Miscellaneous Information:
Computed font is shown as "Local file", so it's not a web loaded font
